frontend/ui/widget/pagebrowserwidget.lua

```diff
@@ -613,19 +614,152 @@ function PageBrowserWidget:showTile(grid_idx, page, tile, do_refresh)
+                end,
+                width = plus_minus_width,
+            }
+        },
+        {
+            {
+                text = _("Thumbnail rows"),
+                callback = function() end,
+                align = "left",
+            },
+            {
+                text = "\u{2796}", -- Heavy minus sign
+                enabled_func = function() return self.nb_rows > self.min_nb_rows end,
+                callback = function()
+                    if self:updateNbRows(-1, true) then
+                        self:updateLayout()
+                    end
+                end,
+                width = plus_minus_width,
+            },
+            {
+                text = "\u{2795}", -- Heavy plus sign
+                enabled_func = function() return self.nb_rows < self.max_nb_rows end,
+                callback = function()
+                    if self:updateNbRows(1, true) then
+                        self:updateLayout()
+                    end
+                end,
+                width = plus_minus_width,
+            }
+        },
+        {
+            {
+                text = _("Chapters in bottom ribbon"),
+                callback = function() end,
+                align = "left",
+            },
+            {
+                text = "\u{2796}", -- Heavy minus sign
+                enabled_func = function() return self.nb_toc_spans > 0 end,
+                callback = function()
+                    if self:updateNbTocSpans(-1, true) then
+                        self:updateLayout()
+                    end
+                end,
+                width = plus_minus_width,
+            },
+            {
+                text = "\u{2795}", -- Heavy plus sign
+                enabled_func = function() return self.nb_toc_spans < self.max_toc_depth end,
+                callback = function()
+                    if self:updateNbTocSpans(1, true) then
+                        self:updateLayout()
+                    end
+                end,
+                width = plus_minus_width,
+            }
+        },
+    }
+    button_dialog = ButtonDialog:new{
+        -- width = math.floor(Screen:getWidth() / 2),
+        width = math.floor(Screen:getWidth() * 0.9), -- max width, will get smaller
+        shrink_unneeded_width = true,
+        buttons = buttons,
+        anchor = function()
+            return self.title_bar.left_button.image.dimen
+        end,
+    }
+    UIManager:show(button_dialog)
+end
+
+function PageBrowserWidget:showAbout()
     UIManager:show(InfoMessage:new{
         text = _([[
 Page browser shows thumbnails of pages.
 
-The bottom ribbon displays an extract of the book map around the shown pages: see the book map help for details.
+The bottom ribbon displays an extract of the book map around the pages displayed:
+
+If statistics are enabled, black bars are shown for already read pages (gray for pages read in the current reading session). Their heights vary with the time spent reading the page.
```

Comment:
depending on

Reply:
full sentence please :)
(I didn't like "vary", so open for a full better sounding sentence.)

Reply:
I think vary sounds alright when combined with depending on, is what I intended to say. :-)

> Their heights vary depending on the time spent reading the page.